pkg/minikube/cluster/cluster.go

```diff
@@ -39,11 +39,13 @@ import (
 	"github.com/minishift/minishift/pkg/minishift/registration"
 	minishiftUtil "github.com/minishift/minishift/pkg/minishift/util"
 	"github.com/minishift/minishift/pkg/util"
+	"github.com/pkg/errors"
```

Comment:
@LalatenduMohanty I saw that this package give nice functions top of standard `errors` package. 

Also, it seems to get installed as part of other dependent package.